### PR TITLE
[DC-1241] Update -1 values for pain PPI question CR to new Base Class

### PR DESCRIPTION
**Pull request by ratuagga**
Negative-ppi CR moved to new Base Class CR.

### CODE REVIEW

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/update_ppi_negative_pain_level_test.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+Integration test for updating answers where the participant skipped answering but the answer was registered as -1.
+
+Original Issue: DC-536
+"""
+# Python imports
+import os
+
+# Third party imports
+from google.cloud import bigquery
+
```
Comment (jp3477) [inline tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/update_ppi_negative_pain_level_test.py:11]: Your integration tests are failing due to lack of date formatting in `tables_and_counts`. You can add this import and reference examples in other integration tests.

```suggestion
from dateutil import parser
```

**Review by jp3477 — CHANGES_REQUESTED** [review]
Added a few suggestions

File: data_steward/cdr_cleaner/cleaning_rules/update_ppi_negative_pain_level.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        value_as_string, and value_source_value that matches record with observation_source_concept_id is 1585747 and
+        value_as_number is -1.
+
+        :return: a list of queries to execute
+        """
+        queries_list = []
+
+        # Select all records where observation_source_concept_id = 1585747 and value_as_number = -1
+        queries_list.append({
+            cdr_consts.QUERY:
```
[the remark this answers is not in the document]
Comment (ratuagga) [reply data_steward/cdr_cleaner/cleaning_rules/update_ppi_negative_pain_level.py:85]: Thanks Jason.. I added.

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/update_ppi_negative_pain_level_test.py (hunk 1 above)
Comment (ratuagga) [reply tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/update_ppi_negative_pain_level_test.py:12]: Done.

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/update_ppi_negative_pain_level_test.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+            'sandboxed_ids': [1, 3],
+            'fields': [
+                'observation_id', 'person_id', 'observation_concept_id',
+                'observation_date', 'observation_type_concept_id',
+                'value_as_number', 'value_as_string', 'value_as_concept_id',
+                'observation_source_concept_id', 'value_source_concept_id',
+                'value_source_value'
+            ],
+            'cleaned_values': [
+                (1, 1, 0, '2020-01-01', 1, None, 'PMI_Skip', 903096, 1585747,
```
[the remark this answers is not in the document]
Comment (ratuagga) [reply tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/update_ppi_negative_pain_level_test.py]: Done

**Review by jp3477 — CHANGES_REQUESTED** [review]
Hi Ratul, thanks for making those changes. I've added another comment about a line that's causing the cleaning rule to fail. Additionally, there are some linting errors that you'll want to check out.

**Review by jp3477 — APPROVED** [review]
Looks great! Thanks for making those changes

**Pull request merged**